### PR TITLE
Added marketplace listing non-citizen upcharge

### PR DESCRIPTION
**Pull request by namedotget**
- Marketplace listings cost 10% more for non-citizens
- Added token value truncation
- Rounded vmooney value on citizen profile

### CODE REVIEW

**Review by pmoncada — CHANGES_REQUESTED** [review]
Two things here...

1.) We need to communicate when people are listing that non-citizens will get charged 10% more.
2.) Please add a test to demonstrate the new feature!

**Comment by namedotget** [comment]
> Two things here...
> 
> 1.) We need to communicate when people are listing that non-citizens will get charged 10% more. 2.) Please add a test to demonstrate the new feature!

In the 2nd commit of this PR I added a message to the TeamMarketplaceListingModal

File: ui/lib/subscription/getMarketplaceListings.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { Arbitrum, Sepolia } from '@thirdweb-dev/chains'
```
Comment (pmoncada) [inline ui/lib/subscription/getMarketplaceListings.ts:1]: Nice! Could you also add a test for the expiration? Add a listing w/ an expiration and make sure that it renders as expired.

File: ui/cypress/e2e/marketplace.cy.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import getMarketplaceListings from '@/lib/subscription/getMarketplaceListings'
```
Comment (pmoncada) [inline ui/cypress/e2e/marketplace.cy.ts:1]: Could we also test the inverse? E.g. a signed in Citizen should see this without the upcharge

**Review by pmoncada — CHANGES_REQUESTED** [review]

**Review by pmoncada — APPROVED** [review]
Nice! Awesome that the privy provider is working

**Pull request merged**